WordPress/gutenberg
# Legacy widget's preview functionality is broken when the page is moved

## Description
Legacy Widget block has a preview function, however, it uses a relative URL.
`iframe` cannot load the widgets.php file when the code is no longer running from the /wp-admin/ URL.
Fixes #30049

## How has this been tested?
1. Go to Appearance -> Widgets page.
2. Click on the "plus sign" button at the very bottom of the page.
3. Add a `Legacy Widget` block.
4. Select `Meta` widget from the dropdown.
5. Click somewhere on the background of the page (not on the `Meta` widget itself). You should see a preview of the `Meta` widget now.
6. Open your browser's developer tools and find an `<iframe />` element with the `wp-block-legacy-widget__edit-preview-iframe`  class. Check its `src` attribute.

#### Expected result:
The `src` attribute of the `<iframe />` should contain an absolute URL.
Please see the screenshots section for reference.

## Screenshots 
![Screenshot 2021-08-12 at 20 00 39](https://user-images.githubusercontent.com/43744263/129246203-e3734aba-d1fc-4ec1-b8e8-6d381e3d488d.png)
![Screenshot 2021-08-12 at 19 50 44](https://user-images.githubusercontent.com/43744263/129246272-05a8fb75-783c-4dbc-9e9a-593a7cd96783.png)


## Types of changes
Bug fix.

## Checklist:
